### DISCUSSION TITLE
Status of FY24 data?

### DISCUSSION THREAD

**Discussion by kevinwolz** [ncss-tech/soilDB#332] [Q&A] [answered]
A few questions about the statuses of things, in case you all know...

- Is SDA now fully on FY24 data?
- Does SDA now contain RSS data?
- Is the SoilWeb WCS fully on FY24 data?
- Do FY24 gSSURGO and gNATSGO mukey grids exist?
- Do FY24 gSSURGO and gNATSGO tabular CSVs exist?

**Comment by dylanbeaudette** [comment]
> A few questions about the statuses of things, in case you all know...
> 
> * Is SDA now fully on FY24 data?

SDA is always updated with the latest the first week of October. Most of SoilWeb is usually updated within the first 2 weeks of October.

> * Does SDA now contain RSS data?

Not yet, but should be "soon".

> * Is the SoilWeb WCS fully on FY24 data?

Not until the FY24 gSSURGO and gNATSGO mukey grids are finished. Should be soon.


> * Do FY24 gSSURGO and gNATSGO mukey grids exist?

Not yet.

> * Do FY24 gSSURGO and gNATSGO tabular CSVs exist?

Not yet.

I'll check on all of these this morning now that I am back at work.

**Comment by brownag** [comment, marked as answer]
@kevinwolz RE: questions 4 and 5

The FY2024 gNATSGO and gSSURGO mukey grids and tabular CSVs are now available here: https://nrcs.app.box.com/v/soils/folder/243464012908

**Reply by kevinwolz** [reply]
Amazing! Thanks for letting me know!

**Comment by kevinwolz** [comment]
Hi All, I noticed that FY2025 data is now mostly live. However, while I see the gNATSGO and gSSURGO mukey grids available [HERE](https://nrcs.app.box.com/v/soils/folder/243464012908), I do not see the tabular CSVs that normally accompany them. Do you know if those will still be produced this year?

**Reply by kevinwolz** [reply]
It looks like the gSSURGO CSVs have been uploaded. I will hold tight for the gNATSGO CSVs!

**Reply by brownag** [reply]
Great Kevin!

In the future, for these types of non-soilDB requests, it is probably best to follow up with the Soils Hotline <soilshotline@usda.gov>. That way National staff know you are interested in these products, and the request can be routed and tracked. While Dylan and I can certainly ask for these types of things on your behalf, it is probably best to hear it from the user.

**Reply by kevinwolz** [reply]
Got it! Thanks, I was not aware of that hotline.
